src/XRP/xrp-error.ts

```diff
@@ -2,6 +2,7 @@
  * Types of errors that originate from interacting with XRPL.
  */
 export enum XRPErrorType {
+  InvalidInputs,
```

Comment:
I would make this singular, unless you really feel like it needs to be plural. 

I've generally seen this be the case in other codebases, and it seems like if you Google it most people agree to use singular: https://stackoverflow.com/questions/15755955/naming-of-enums-in-java-singular-or-plural